cloud_service/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -0,0 +1,97 @@
+:_content-type: ASSEMBLY
+[id="getting-started-rhacs-cloud-ocp"]
+= Getting started with RHACS Cloud Service
+include::modules/common-attributes.adoc[]
+:context: getting-started-rhacs-cloud-ocp
+
+toc::[]
+
+[role="_abstract"]
+
+{product-title-managed} ({product-title-managed-short}) provides security services for your {cloud-family} and Kubernetes clusters. This topic provides instructions for using {product-title-managed-short} to secure clusters that are running on supported platforms. See xref:../installing/acs-installation-platforms.adoc#acs-installation-platforms[Supported platforms and installation methods] for more information on supported platforms for secured clusters.
+
+:FeatureName: {product-title-managed-short}
+include::snippets/field-trial.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* Ensure that you can access the *Advanced Cluster Security* menu option from the {cloud-console}.
++
+[NOTE]
+====
+To access the {product-title-managed-short} console, you need your Red Hat Single Sign-On (RH-SSO) credentials, or credentials for another identity provider if that has been configured. See xref:../cloud_service/getting-started-rhacs-cloud-ocp.adoc#default-access-acs-console[Default access to the ACS console].
+====
+
+[id="installation-overview-acs-cloud"]
+== High-level overview of installation steps
```

Comment:
```suggestion
== Overview of installation steps
```

Reply:
This language matches the headings in other sections (for example, /installing/installing_ocp/install-rhacs-ocp.adoc) - and it is a high-level overview, as opposed to detailed (because the details are provided in the links), so I am going to leave as is.

Reply:
:+1: